pkg/storage/testdata/mvcc_histories/export_fingerprint_no_timestamps

```diff
@@ -0,0 +1,77 @@
+# Test MVCC Fingerprinting when configured to ignore timestamps
+#
+# This test ensures that when the stripTimestamp flag is applied to a fingerpint export request, the
+# timestamps in the MVCC history of a key span are ignored. To test, a simple history is
+# constructed, fingerprinted, and wiped. Then the same history is reconstructed with the timestamps
+# shifted up.
+
+# Sets up the following dataset, where x is MVCC point tombstone, o-o is MVCC
+# range tombstone, [] is intent.  We include some local timestamps, which should
+# not be export fingerprinted.
+#
+#  6
+#  5      x   o---o
+#  4  a2  b1  o------o
+#  -----------------------------------------
+#  3          o---o
+#  2  a2  x
+#  1      b1  o-------o
+#     a   b   c   d   e
+
+
+run ok
+put k=a ts=2 v=a2
```

Comment:
NB: If you want to test the table prefix stripping as well, if you do something like `put k=/b ts=2 v=b tenant-prefix=10 init-checksum` you'll get a key that looks like a table key.